internal/utils/string_shortener.go

```diff
@@ -0,0 +1,22 @@
+package utils
+
+import (
+	"errors"
+	"math/rand"
+)
+
+const (
+	ShortenLength = 8
+	charset       = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+)
+
+func Shorten(s string) (string, error) {
+	if len(s) == 0 {
+		return "", errors.New("string should not be empty")
```

Comment:
[SUGGESTION] It would be better if it was a variable

```
var ErrOutOfTea = fmt.Errorf("no more tea available")
var ErrPower = fmt.Errorf("can't boil water")
```